src/main/java/org/breedinginsight/utilities/BrAPIDAOUtil.java

```diff
@@ -273,6 +276,15 @@ private <T, V> List<V> getListResult(ApiResponse<Pair<Optional<T>, Optional<BrAP
                 new ArrayList<>();
     }
 
+    // TODO: write generic put code
+    public <T> List<T> put(List<T> brapiObjects,
+                           ImportUpload upload,
+                           Function<List<T>, ApiResponse> putMethod,
+                           Consumer<ImportUpload> progressUpdateMethod) throws ApiException {
+
```

Comment:
If this isn't implemented, you may want to have this method throw an `UnsupportedOperationException` for now

Reply:
Added `UnsupportedOperationException`